src/main/features/claim/form/models/eligibility/over18Defendant.ts

```diff
@@ -0,0 +1,27 @@
+import { YesNoOption } from 'models/yesNoOption'
+import { PartyType } from 'app/common/partyType'
+
+export class Over18Defendant {
+
+  static readonly YES = new Over18Defendant(YesNoOption.YES.option)
+  static readonly NO = new Over18Defendant(YesNoOption.NO.option)
+  static readonly COMPANY_OR_ORGANISATION = new Over18Defendant(PartyType.ORGANISATION.value)
+
+  constructor (public readonly option: string) {
+  }
+
+  static fromObject (input?: any): Over18Defendant {
+    if (!input) {
+      return input
+    }
+    return this.all().filter(claimValue => claimValue.option === input).pop()
```

Comment:
Please rename `claimValue`, iterated item has nothing to do with claim value, I guess copy / paste happened here.